components/thumbnail/basic/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+lib
+node_modules
```

Comment:
"sui-studio generate" creates a general configuration. As sui-studio-create does not add gitignore, "sui-studio generate" can't assume you already added node_modules to the root folder.
But, if the root folder has a .gitignore, it makes sense you don't commit duplicated instruction in every single component.